### ISSUE TITLE
Tensorflow Addon FBetaScore metric error: "Got invalid value [0.13744274] for metric 'fbeta_score'"

### ISSUE THREAD

**Issue by zoltan-fedor** [mlflow/mlflow#4183]
I am using MLFlow 2.4.1 with Tensorflow 2.3.1 and MLFlow autologging and all worked well with the standard metrics.
But then I decided to try the `FBetaScore` metric from the tensorflow-addons package (see https://www.tensorflow.org/addons/api_docs/python/tfa/metrics/FBetaScore) and which causes the following warning:

`2021/03/16 19:43:38 WARNING mlflow.utils.autologging_utils: MLflow issued a warning during tensorflow autologging: ".../lib/python3.8/site-packages/mlflow/utils/autologging_utils.py:242: UserWarning: Logging to MLflow failed: Got invalid value [0.13744274] for metric 'fbeta_score' (timestamp=1615938218333). Please specify value as a valid double (64-bit floating point)"`

It seems as if the value for the metric is a list (or a Tensor) and MLFlow only accepts direct numerical values, see https://github.com/mlflow/mlflow/blob/3935ad1cd9cb556ac931de9ed5c7c9538a3a8281/mlflow/utils/validation.py#L72

Any ideas how to work around this?

My code:

```
import tensorflow as tf
from tensorflow.keras import layers, metrics
import tensorflow_addons as tfa
import mlflow.tensorflow

...

mlflow.tensorflow.autolog(every_n_iter=1)

...

_metrics = [
    'accuracy',
    metrics.TruePositives(name='tp'),
    metrics.FalsePositives(name='fp'),
    metrics.TrueNegatives(name='tn'),
    metrics.FalseNegatives(name='fn'),
    metrics.Precision(name='precision'),
    metrics.Recall(name='recall'),
    metrics.AUC(name='auc'),
    # calculating F2 using https://www.tensorflow.org/addons/api_docs/python/tfa/metrics/FBetaScore
    tfa.metrics.FBetaScore(num_classes=1, beta=2.0,
                           threshold=0.9999999,
                           name='fbeta_score'
                           )
]

model.compile(optimizer=optimizer,
              loss=tf.keras.losses.BinaryCrossentropy(from_logits=True),
              metrics=_metrics)

early_stopping = tf.keras.callbacks.EarlyStopping(
    monitor='fbeta_score',
    verbose=1,
    patience=10,
    mode='max',
    restore_best_weights=True)

history = model.fit(train_ds,
                    validation_data=val_ds,
                    epochs=max_epochs,
                    batch_size=batch_size,
                    class_weight={0: 1, 1: 10},
                    callbacks=[early_stopping],
                    verbose=0
                    )
```

**Comment by zoltan-fedor** [comment]
The problem was that I have tried to use the `FBetaScore()` without the `average` param.

```
tfa.metrics.FBetaScore(num_classes=2, beta=2.0,
                               average='micro',
                               threshold=0.9999999,
                               name='fbeta_score'
                               )
```
